example/App.js

```diff
@@ -34,21 +34,40 @@ export default class App extends Component<Props> {
     this.show('time');
   }
 
+  mmddyyyy = (date) => {
+    var yyyy = date.getUTCFullYear();
+    var mm = date.getUTCMonth() < 9 ? '0' + (date.getUTCMonth() + 1) : (date.getUTCMonth() + 1); // getUTCMonth() is zero-based
```

Comment:
This can probably be cleaned up a bit if the optional `0` is prepended after the actual, also would this display `010` if the returned month is `9`? Also would prefer regular functions here as part of the `class`

Reply:
As this is an exampe app, can't we just use `data-fn`'s to clean this up a little bit?

Reply:
I have ended up using `moment`. `date-fns` was giving me problems for timezones and it seems that there is no way of defining a timezone for the format function.